src/stretch/agent/operations/grasp_object.py

```diff
@@ -918,9 +918,16 @@ def run(self) -> None:
 
         # Compute the angles necessary
         if self.use_pitch_from_vertical:
-            head_pos = obs.camera_pose[:3, 3]
-            dy = np.abs(head_pos[1] - relative_object_xyz[1])
-            dz = np.abs(head_pos[2] - relative_object_xyz[2])
+            # head_pos = obs.camera_pose[:3, 3]
```

Comment:
Why do we care about head pos when we use visual servo manipulation?

Reply:
its for computing angle but you're probably right